app/code/Magento/Checkout/view/frontend/web/js/product-prices-updated.js

```diff
@@ -8,22 +8,21 @@ define([
     'Magento_Customer/js/customer-data'], function ($, customerData) {
     'use strict';
 
-    return function () {
+    return function (data) {
         var cacheKey = 'cart-cache-updated',
             cartData = customerData.get('cart'),
-            localStorage = $.initNamespaceStorage('mage-cache-storage').localStorage,
-            cartCacheUpdated = localStorage.get(cacheKey);
+            cartCacheUpdated = window.localStorage.getItem(cacheKey);
```

Comment:
please add try-catch for case when localstorage not availabe, similar to 
https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/app/code/Magento/Catalog/view/frontend/web/js/product/storage/ids-storage.js#L14-L27

BTW seems like we have in multiple places the "localstorage" implementation, maybe it would be good to create such library, and use it in all places, similar to https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/app/code/Magento/Ui/view/base/web/js/lib/core/storage/local.js#L21-L79